### PR TITLE
Refactor/home my services

### PR DESCRIPTION
**Pull request by marie-j**
## Related 
- [ ] https://github.com/ovh-ux/ovh-module-exchange/pull/314

### CODE REVIEW

File: client/app/billing/autoRenew/actions/activation/activation.controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
```
Comment (frenautvh) [inline client/app/billing/autoRenew/actions/activation/activation.controller.js]: should we just import get from lodash ?

Comment (marie-j) [reply client/app/billing/autoRenew/actions/activation/activation.controller.js]: Still lodash3 :/

**Comment by ovh-cds** [comment]
CDS Report build-manager#2254.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✔

File: client/app/user/User.class.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { AUTORENEW_RECENT_SUBSIDIARIES } from './user.constants';
+
+export default class {
+  constructor(user) {
+    Object.assign(this, user);
+  }
+
+  hasRecentAutorenew() {
```
Comment (cbourgois) [inline client/app/user/User.class.js]: `recent` for now, not in 1y.

**Pull request merged**